index.bs

```diff
@@ -10796,11 +10795,30 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				[=Computing a block of audio|compute a block of audio=], and
 				[=Making a buffer available for reading|make it available for reading=].
 
-			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
-				{{AudioWorkletProcessor}}'s {{AudioWorkletProcessor/process()}} method
-				to [=Computing a block of audio|compute a block of audio=] with
-				the argument of [=input buffer=], output buffer and
-				[=input AudioParam buffer=].
+			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, execute these steps:
+
+			  1. If <a>callable process</a> is `false`, abort the following sub steps.
+
+				1. Let <var>processor</var> be the associated {{AudioWorkletProcessor}}
+					instance.
+
+				1. Let <var>processFunction</var> be the result of
+					<code><a href="http://www.ecma-international.org/ecma-262/6.0/#sec-get-o-p">
+					Get</a>(O=<i><var>processor</var></i>, P="process")</code>.
+
+				1. If <code><a href="http://www.ecma-international.org/ecma-262/6.0/#sec-iscallable">
+					IsCallable</a>(argument=<i><var>processFunction</var></i>)</code> is
+					false, execute the following:
+
+					1. Queue a control message to fire `onprocessorerror` event on the
+						associated AudioWorkletNode.
+
+					1. Set <a>callable process</a> to `false`.
```

Comment:
The algorithm flow is to continue after setting this to false.  Is that what you want?  I think not since `IsCallable` returned false, you shouldn't call the `process` method in the next step.